pkg/skaffold/runner/v1/deploy.go

```diff
@@ -103,15 +103,7 @@ See https://skaffold.dev/docs/pipeline-stages/taggers/#how-tagging-works`)
 	ctx, endTrace := instrumentation.StartTrace(ctx, "Deploy_Deploying")
 	defer endTrace()
 
-	// we only want to register images that are local AND were built by this runner
-	var localAndBuiltImages []graph.Artifact
-	for _, image := range localImages {
-		if r.wasBuilt(image.Tag) {
-			localAndBuiltImages = append(localAndBuiltImages, image)
-		}
-	}
-
-	r.deployer.RegisterLocalImages(localAndBuiltImages)
+	r.deployer.RegisterLocalImages(localImages)
```

Comment:
@nkubala, can you take a look at this change? I remember there was some reason why you added this logic.

Reply:
I think I added this logic to make sure we don't try and load images from a different pipeline when we have multiple configs. it seems like the issue here is that we only take into account the cluster location when determining whether we should load an image, but sometimes users might want to force an image load.

Reply:
Two things:

1) We need a good way to allow user to force load an image to a local cluster, preferably via CLI flag (will create issue)
2) Integration tests are not recognizing kind as local cluster (need to debug)